### PR TITLE
removing role identifiers for ansible 1.4

### PR DESCRIPTION
**Pull request by jarv**

### CODE REVIEW

**Comment by e0d** [comment]
Can you update the templates in ansible-role:

```
- name: {{ role_name }} | stub ansible task
```

Other than that LGTM.

File: playbooks/roles/gluster/tasks/main.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 # Install and configure simple glusterFS shared storage
 
-- name: gluster | all | Install common packages
```
Comment (feanil) [inline playbooks/roles/gluster/tasks/main.yml:4]: The all/primary part should probably be left in for Gluster.

**Comment by feanil** [comment]
Looks good other than the two comments above.

File: playbooks/roles/gluster/tasks/main.yml (hunk 1 above)
Comment (jarv) [inline playbooks/roles/gluster/tasks/main.yml:4]: I blame sed for not making non-greedy substitutions easy

Comment (carsongee) [inline playbooks/roles/gluster/tasks/main.yml:4]: It is one of those annoying times I feel forced to use perl

**Comment by jarv** [comment]
@carsongee @e0d @feanil I've verified a from-scratch install and this checks out.
OK to merge?

**Comment by carsongee** [comment]
:+1:

**Pull request merged**